src/ols_regressor/regressor.py

```diff
@@ -94,9 +83,10 @@ def predict(self, X):
         if not np.isfinite(X).all():
             raise ValueError("Input contains infinite values.")
 
-        pred = X @ self.coef
+        pred = X @ self.coef[1:]
```

Comment:
I think we should also add a column of one for the prediction and use the full coef matrix

Reply:
Added this in. It reduces our model score though. But I think mathematically it makes more sense to include it.